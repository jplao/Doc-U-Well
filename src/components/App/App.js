import React, { Component } from 'react';
import { Route, Switch, Link, NavLink } from 'react-router-dom';
import '../../index.scss';
import Home from '../../containers/Home/Home';
import Display from '../../containers/Display/Display';



class App extends Component {
// <i class="fas fa-user-md"></i>
// <i class="far fa-address-card"></i>
// <i class="far fa-calendar-alt"></i>
// <i class="fas fa-user-circle"></i>
// <i class="fas fa-trash-alt"></i>
// <i class="fas fa-edit"></i>
// <i class="far fa-plus-square"></i>


  render() {
    return (
      <section className="main-container">
        <header className="header-container">
          <Link to='/' className='link-header'>
            <h1>My Health Navigator</h1>
          </Link>
        </header>
        <Switch>
          <Route exact path='/' component={Home} />
          <Route exact path='/profile' render={() => <Display type="profile" />} />
          <Route exact path='/appointments' render={() => <Display type="appointments" />} />
          <Route exact path='/providers' render={() => <Display type="providers" />} />
          <Route exact path='/insurance' render={() => <Display type="insurance" />} />
          <Route render={() => (
            <div className='redirect'>
              <Link to='/' className='link-redirect'>
                <h2>Cannot find that page</h2>
                <button className='home'> Click to Return Home</button>
              </Link>
            </div>
            )}  />
        </Switch>
        <nav className='nav-container'>
          <NavLink to='/profile' className='navigation'>
            <div>
              <i className="fas fa-user-circle navigation"></i>
              <p>Profile</p>
            </div>
          </NavLink>
          <NavLink to='/appointments' className='navigation'>
            <div>
              <i className="far fa-calendar-alt navigation"></i>
              <p>Appointments</p>
            </div>
          </NavLink>
          <NavLink to='/providers' className='navigation'>
            <div>
              <i className="fas fa-user-md navigation"></i>
              <p>Providers</p>
            </div>
          </NavLink>
          <NavLink to='/insurance' className='navigation'>
            <div>
              <i className="far fa-address-card navigation"></i>
              <p>Insurance</p>
              </div>
          </NavLink>
          </nav>
      </section>
    );
  }
}

export default App;
