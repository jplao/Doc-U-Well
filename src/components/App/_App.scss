.link-header{
  color: #301E0A;
  text-decoration: none;
}

.main-container {
  background-color: #314A4f;
  height: 100vh;
  display: grid;
  grid-template-columns: 1fr;
  justify-items: center;
  align-items: center;
  grid-template-rows: 1fr 6fr 1fr;
}

.header-container{
  background-color: #9ec65d;
  height: 100%;
  width: 100%;
  display: grid;
  grid-template-columns: 1fr;
  justify-items: center;
  align-items: center;
}

.navigation{
  color: #fdf2ea;
  padding: 10px;
  text-decoration: none;
  &:hover{
    background-color: #9ec65d;
  }
  &:active{
    background-color: #fdf2ea;
    color: #301e0a;
  }
}

.nav-container{
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  text-align: center;
  font-size: .75rem;

}

.fas{
  color: #fdf2ea;
  font-size: 1rem;
}

.far{
  color: #fdf2ea;
  font-size: 1rem;
}
