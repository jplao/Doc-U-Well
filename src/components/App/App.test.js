import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App Component', () => {
  it('should match the screenshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()

  })
})
