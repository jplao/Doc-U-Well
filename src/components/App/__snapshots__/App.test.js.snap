// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match the screenshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="header-container"
        >
          <Link
            className="link-header"
            replace={false}
            to="/"
          >
            <h1>
              My Health Navigator
            </h1>
          </Link>
        </header>,
        <Switch
          className="middle-container"
        >
          <Route
            component={[Function]}
            exact={true}
            path="/"
          />
          <Route
            exact={true}
            path="/profile"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/appointments"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/providers"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/insurance"
            render={[Function]}
          />
          <Route
            render={[Function]}
          />
        </Switch>,
        <nav
          className="nav-container"
        >
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="navigation"
            to="/profile"
          >
            <div>
              <i
                className="fas fa-user-circle navigation"
              />
              <p>
                Profile
              </p>
            </div>
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="navigation"
            to="/appointments"
          >
            <div>
              <i
                className="far fa-calendar-alt navigation"
              />
              <p>
                Appointments
              </p>
            </div>
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="navigation"
            to="/providers"
          >
            <div>
              <i
                className="fas fa-user-md navigation"
              />
              <p>
                Providers
              </p>
            </div>
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="navigation"
            to="/insurance"
          >
            <div>
              <i
                className="far fa-address-card navigation"
              />
              <p>
                Insurance
              </p>
            </div>
          </NavLink>
        </nav>,
      ],
      "className": "main-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className="link-header"
            replace={false}
            to="/"
          >
            <h1>
              My Health Navigator
            </h1>
          </Link>,
          "className": "header-container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <h1>
              My Health Navigator
            </h1>,
            "className": "link-header",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "My Health Navigator",
            },
            "ref": null,
            "rendered": "My Health Navigator",
            "type": "h1",
          },
          "type": [Function],
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Route
              component={[Function]}
              exact={true}
              path="/"
            />,
            <Route
              exact={true}
              path="/profile"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/appointments"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/providers"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/insurance"
              render={[Function]}
            />,
            <Route
              render={[Function]}
            />,
          ],
          "className": "middle-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/profile",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/appointments",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/providers",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/insurance",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/profile"
            >
              <div>
                <i
                  className="fas fa-user-circle navigation"
                />
                <p>
                  Profile
                </p>
              </div>
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/appointments"
            >
              <div>
                <i
                  className="far fa-calendar-alt navigation"
                />
                <p>
                  Appointments
                </p>
              </div>
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/providers"
            >
              <div>
                <i
                  className="fas fa-user-md navigation"
                />
                <p>
                  Providers
                </p>
              </div>
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/insurance"
            >
              <div>
                <i
                  className="far fa-address-card navigation"
                />
                <p>
                  Insurance
                </p>
              </div>
            </NavLink>,
          ],
          "className": "nav-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <div>
                <i
                  className="fas fa-user-circle navigation"
                />
                <p>
                  Profile
                </p>
              </div>,
              "className": "navigation",
              "to": "/profile",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="fas fa-user-circle navigation"
                  />,
                  <p>
                    Profile
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-user-circle navigation",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Profile",
                  },
                  "ref": null,
                  "rendered": "Profile",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <div>
                <i
                  className="far fa-calendar-alt navigation"
                />
                <p>
                  Appointments
                </p>
              </div>,
              "className": "navigation",
              "to": "/appointments",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="far fa-calendar-alt navigation"
                  />,
                  <p>
                    Appointments
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-calendar-alt navigation",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Appointments",
                  },
                  "ref": null,
                  "rendered": "Appointments",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <div>
                <i
                  className="fas fa-user-md navigation"
                />
                <p>
                  Providers
                </p>
              </div>,
              "className": "navigation",
              "to": "/providers",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="fas fa-user-md navigation"
                  />,
                  <p>
                    Providers
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-user-md navigation",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Providers",
                  },
                  "ref": null,
                  "rendered": "Providers",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <div>
                <i
                  className="far fa-address-card navigation"
                />
                <p>
                  Insurance
                </p>
              </div>,
              "className": "navigation",
              "to": "/insurance",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <i
                    className="far fa-address-card navigation"
                  />,
                  <p>
                    Insurance
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "far fa-address-card navigation",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Insurance",
                  },
                  "ref": null,
                  "rendered": "Insurance",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
        ],
        "type": "nav",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="header-container"
          >
            <Link
              className="link-header"
              replace={false}
              to="/"
            >
              <h1>
                My Health Navigator
              </h1>
            </Link>
          </header>,
          <Switch
            className="middle-container"
          >
            <Route
              component={[Function]}
              exact={true}
              path="/"
            />
            <Route
              exact={true}
              path="/profile"
              render={[Function]}
            />
            <Route
              exact={true}
              path="/appointments"
              render={[Function]}
            />
            <Route
              exact={true}
              path="/providers"
              render={[Function]}
            />
            <Route
              exact={true}
              path="/insurance"
              render={[Function]}
            />
            <Route
              render={[Function]}
            />
          </Switch>,
          <nav
            className="nav-container"
          >
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/profile"
            >
              <div>
                <i
                  className="fas fa-user-circle navigation"
                />
                <p>
                  Profile
                </p>
              </div>
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/appointments"
            >
              <div>
                <i
                  className="far fa-calendar-alt navigation"
                />
                <p>
                  Appointments
                </p>
              </div>
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/providers"
            >
              <div>
                <i
                  className="fas fa-user-md navigation"
                />
                <p>
                  Providers
                </p>
              </div>
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="navigation"
              to="/insurance"
            >
              <div>
                <i
                  className="far fa-address-card navigation"
                />
                <p>
                  Insurance
                </p>
              </div>
            </NavLink>
          </nav>,
        ],
        "className": "main-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="link-header"
              replace={false}
              to="/"
            >
              <h1>
                My Health Navigator
              </h1>
            </Link>,
            "className": "header-container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <h1>
                My Health Navigator
              </h1>,
              "className": "link-header",
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "My Health Navigator",
              },
              "ref": null,
              "rendered": "My Health Navigator",
              "type": "h1",
            },
            "type": [Function],
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Route
                component={[Function]}
                exact={true}
                path="/"
              />,
              <Route
                exact={true}
                path="/profile"
                render={[Function]}
              />,
              <Route
                exact={true}
                path="/appointments"
                render={[Function]}
              />,
              <Route
                exact={true}
                path="/providers"
                render={[Function]}
              />,
              <Route
                exact={true}
                path="/insurance"
                render={[Function]}
              />,
              <Route
                render={[Function]}
              />,
            ],
            "className": "middle-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "component": [Function],
                "exact": true,
                "path": "/",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "exact": true,
                "path": "/profile",
                "render": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "exact": true,
                "path": "/appointments",
                "render": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "exact": true,
                "path": "/providers",
                "render": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "exact": true,
                "path": "/insurance",
                "render": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "render": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="navigation"
                to="/profile"
              >
                <div>
                  <i
                    className="fas fa-user-circle navigation"
                  />
                  <p>
                    Profile
                  </p>
                </div>
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="navigation"
                to="/appointments"
              >
                <div>
                  <i
                    className="far fa-calendar-alt navigation"
                  />
                  <p>
                    Appointments
                  </p>
                </div>
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="navigation"
                to="/providers"
              >
                <div>
                  <i
                    className="fas fa-user-md navigation"
                  />
                  <p>
                    Providers
                  </p>
                </div>
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="navigation"
                to="/insurance"
              >
                <div>
                  <i
                    className="far fa-address-card navigation"
                  />
                  <p>
                    Insurance
                  </p>
                </div>
              </NavLink>,
            ],
            "className": "nav-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <div>
                  <i
                    className="fas fa-user-circle navigation"
                  />
                  <p>
                    Profile
                  </p>
                </div>,
                "className": "navigation",
                "to": "/profile",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="fas fa-user-circle navigation"
                    />,
                    <p>
                      Profile
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fas fa-user-circle navigation",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Profile",
                    },
                    "ref": null,
                    "rendered": "Profile",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <div>
                  <i
                    className="far fa-calendar-alt navigation"
                  />
                  <p>
                    Appointments
                  </p>
                </div>,
                "className": "navigation",
                "to": "/appointments",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="far fa-calendar-alt navigation"
                    />,
                    <p>
                      Appointments
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "far fa-calendar-alt navigation",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Appointments",
                    },
                    "ref": null,
                    "rendered": "Appointments",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <div>
                  <i
                    className="fas fa-user-md navigation"
                  />
                  <p>
                    Providers
                  </p>
                </div>,
                "className": "navigation",
                "to": "/providers",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="fas fa-user-md navigation"
                    />,
                    <p>
                      Providers
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "fas fa-user-md navigation",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Providers",
                    },
                    "ref": null,
                    "rendered": "Providers",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <div>
                  <i
                    className="far fa-address-card navigation"
                  />
                  <p>
                    Insurance
                  </p>
                </div>,
                "className": "navigation",
                "to": "/insurance",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="far fa-address-card navigation"
                    />,
                    <p>
                      Insurance
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "far fa-address-card navigation",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Insurance",
                    },
                    "ref": null,
                    "rendered": "Insurance",
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
          ],
          "type": "nav",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
